### PR TITLE
[Doc Improvement][Customizable feedback forms][4132523]

### PR DESCRIPTION
**Pull request by v-kushals**
Duplicate of #11185.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bot-messages-ai-generated-content.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1a3b53550f4bffc970873160ef81b36f1b24a6c6/msteams-platform%2Fbots%2Fhow-to%2Fbot-messages-ai-generated-content.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aacfb699-c806-4383-8fd7-9cbbbb313c25) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bot-messages-ai-generated-content.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d073dc53ee01a3f28bfceda8fa6d26edad9db7bc/msteams-platform%2Fbots%2Fhow-to%2Fbot-messages-ai-generated-content.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b2cc6ceb-3648-4182-92be-74a7330382ec) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bot-messages-ai-generated-content.md](https://github.com/MicrosoftDocs/msteams-docs/blob/815d154a823424139008c560d324de5b1cdc46b6/msteams-platform%2Fbots%2Fhow-to%2Fbot-messages-ai-generated-content.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e22cb2b9-e5d2-4918-804d-91e56e7974f6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bot-messages-ai-generated-content.md](https://github.com/MicrosoftDocs/msteams-docs/blob/60e2ad2168e37e6766273c30f4db84f79305269e/msteams-platform%2Fbots%2Fhow-to%2Fbot-messages-ai-generated-content.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d477a78-3e59-48c3-b42f-4c38ae65959e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bot-messages-ai-generated-content.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6fa670d0106b9b5f9509fc2d71e076c3aee773bc/msteams-platform%2Fbots%2Fhow-to%2Fbot-messages-ai-generated-content.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9ca05a01-ebe6-4ab9-a285-6bb812464eb3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bot-messages-ai-generated-content.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0b356ba7ba696bcb1519e780fc3c72ecab9d1463/msteams-platform%2Fbots%2Fhow-to%2Fbot-messages-ai-generated-content.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bc29e8f9-c406-48d6-9997-b89d1166af45) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/bot-messages-ai-generated-content.md [hunk 1]
```diff
@@ -175,29 +175,48 @@ export const app = new Application<ApplicationTurnState>({
...
 
 | Property | Type | Required | Description |
 |--|--|--|--|
-| `feedbackLoopEnabled` | Boolean | Yes | Enables feedback buttons in the bot's message |
+| `feedbackLoop` | Object | ✔️ | Enables feedback buttons in the bot's message. |
+| `feedbackLoop.type` | String | ✔️ | Defines the type of feedback form that appears when a user selects the feedback buttons.<br>Allowed values: `custom`, `default` |
+
+If you set `feedbackLoop.type` to `default`, the default feedback form appears when a user selects the feedback buttons. If you set `feedbackLoop.type` to `custom`, the following invoke request is sent to the bot to retrieve a custom form to be displayed to the user:
+
+```JSON
```
Comment (Wajeed-msft) [inline msteams-platform/bots/how-to/bot-messages-ai-generated-content.md:202]: Do we have sample code for this? If not, please reach out to sample code team.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/bot-messages-ai-generated-content.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ad8e35b8480ff7ac6a73d34f4674fb60de79415e/msteams-platform%2Fbots%2Fhow-to%2Fbot-messages-ai-generated-content.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6ff08c58-2809-4f46-910c-566be49b6a27) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/bot-messages-ai-generated-content.md (hunk 1 above)
Comment (v-kushals) [reply msteams-platform/bots/how-to/bot-messages-ai-generated-content.md:202]: Yes, the sample code is being created.

**Pull request closed without merging**